inst/qml/Prophet.qml

```diff
@@ -462,7 +462,7 @@ Form
 					label: qsTr("Horizon")
 					defaultValue: 3
 				}
-				IntegerField
+				DoubleField
```

Comment:
I tested this and while it works, I would probably change it back to an IntegerField and instead set the Horizon option to a even value like 4.

The reason is that the Period between cutoffs defines how many periods the module shifts the training period ahead for cross validation. And if my data is in the hourly format and I set this option to 1.5 days it shifts the series ahead 36 hours which makes sense.

But for some reason prophet also allows me to set it to 1.5 hours if I have hourly data. Technically this doesn't make sense since I don't have any datapoints 1.5 hours ahead that I could fit the data to  - but it still computes.
So I would probably change it back to Integer for this release - and maybe adjust it later once I checked the code of what prophet does internally.

Reply:
> I tested this and while it works, I would probably change it back to an IntegerField and instead set the Horizon option to a even value like 4.
> 
> The reason is that the Period between cutoffs defines how many periods the module shifts the training period ahead for cross validation. And if my data is in the hourly format and I set this option to 1.5 days it shifts the series ahead 36 hours which makes sense.
> 
> But for some reason prophet also allows me to set it to 1.5 hours if I have hourly data. Technically this doesn't make sense since I don't have any datapoints 1.5 hours ahead that I could fit the data to - but it still computes. So I would probably change it back to Integer for this release - and maybe adjust it later once I checked the code of what prophet does internally.

OK, then I have used the Math.floor function.